shared/lib_battery_lifetime_lmolto.h

```diff
@@ -0,0 +1,101 @@
+/**
+BSD-3-Clause
+Copyright 2019 Alliance for Sustainable Energy, LLC
+Redistribution and use in source and binary forms, with or without modification, are permitted provided
+that the following conditions are met :
+1.	Redistributions of source code must retain the above copyright notice, this list of conditions
+and the following disclaimer.
+2.	Redistributions in binary form must reproduce the above copyright notice, this list of conditions
+and the following disclaimer in the documentation and/or other materials provided with the distribution.
+3.	Neither the name of the copyright holder nor the names of its contributors may be used to endorse
+or promote products derived from this software without specific prior written permission.
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES,
+INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ARE DISCLAIMED.IN NO EVENT SHALL THE COPYRIGHT HOLDER, CONTRIBUTORS, UNITED STATES GOVERNMENT OR UNITED STATES
+DEPARTMENT OF ENERGY, NOR ANY OF THEIR EMPLOYEES, BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY,
+OR CONSEQUENTIAL DAMAGES(INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT
+OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*/
+
+#ifndef SAM_SIMULATION_CORE_LIB_BATTERY_LIFETIME_LMOLTO_H
+#define SAM_SIMULATION_CORE_LIB_BATTERY_LIFETIME_LMOLTO_H
+
+#include "lib_util.h"
+#include "lib_battery_lifetime_calendar_cycle.h"
+
+/**
+ * LMO/LTO Life Model
+ *
+ * Based on the model developed by NREL with data from INL:
+ *  Reference after published
+ */
+
+extern const double Rug;
+extern const double T_ref;
+extern const double F;
+
+struct lifetime_lmolto_state {
+    double dq_relative_cal;                 // %
+    double dq_relative_cyc;                 // %
+    double EFC;                             // Equivalent Full Cycles
+
+    // lifetime capacity updated after 24 hours elapsed using below values, which are all reset each day
+    double EFC_dt;                          // 1
```

Comment:
What is the // 1 comment? Shouldn't dt have units of hrs or days?

Reply:
Fixed